beacon-chain/cache/depositsnapshot/deposit_tree.go

```diff
@@ -83,14 +83,15 @@ func fromSnapshot(snapshot DepositTreeSnapshot) (*DepositTree, error) {
 }
 
 // Finalize marks a deposit as finalized.
-func (d *DepositTree) Finalize(eth1data *ethpb.Eth1Data, executionBlockHeight uint64) error {
+func (d *DepositTree) Finalize(eth1DepositIndex int64, executionHash common.Hash) error {
```

Comment:
@nisdas is there a reason why you have switched from using `uint64` to `int64`?

Reply:
The deposit index is `-1` for empty tries, and is represented with `-1` in other places during initialization

Reply:
Thanks for clarifying!